src/citrine/informatics/design_spaces/sample_design_space.py

```diff
@@ -0,0 +1,18 @@
+from citrine._serialization import properties
+from citrine._serialization.serializable import Serializable
+
+
+class SampleDesignSpaceInput(Serializable['SampleDesignSpaceInput']):
```

Comment:
Moved to its own file to remove circular dependency

Reply:
What circular dependency?

Reply:
This was introduced when moving the execution collection from projects.py to design_space.py.